pkg/resource/stack/deployment_test.go

```diff
@@ -498,6 +498,18 @@ func TestSerializePropertyValue(t *testing.T) {
 	})
 }
 
+// Test that if ShowSecrets is set the encrypter is not called into at all.
+func TestSerializePropertyValue_ShowSecrets(t *testing.T) {
+	t.Parallel()
+
+	rapid.Check(t, func(t *rapid.T) {
```

Comment:
Curious why we're making this a property test?  The PanicCrypter would panic with any value, so giving it more different secret values doesn't seem like it adds much to the test, but the test would take longer overall?  Or is there something I'm missing?

Reply:
Oh I just copied the other SerializePropertyValue test. I could change this to just a straight line test of a couple of secret values.